components/Form/Details/index.js

```diff
@@ -1,13 +1,18 @@
 import css from './index.module.scss';
 
 const Details = ({ color, children, title, onclick, id }) => (
-  <details className={`govuk-details ${css['lbh-details']}`} data-module="govuk-details" id={`details-${id}`}>
+  <details
+    className={`govuk-details ${css['lbh-details']}`}
+    data-module="govuk-details"
+    id={`details-${id}`}>
     <summary
       className={`govuk-details__summary ${css['lbh-details__summary']}`}
       style={color && { color: `${color}` }}
       id={`summary-${id}`}
       onClick={onclick}>
-      <span className="govuk-details__summary-text">{title}</span>
+      <span id={`span-${id}`} className="govuk-details__summary-text">
```

Comment:
This is the only line that's changed (added an id to the span) the rest is just formatting